test/Benchmarks/README.md

```diff
@@ -3,5 +3,5 @@
 Use the following example to run Benchmarks from command line:
 (change parameters as necessary)
 
-1. navigate to opentelemetry-dotnet\src\benchmarks directory and run the following
```

Comment:
Minor: I think the document title should be "OpenTelemetry Benchmarks" (1 removing the SDK since it is not specific to SDK 2 plural form for consistency with the project/folder name)

Reply:
removed SDK and updated to plural form